app/assets/javascripts/discourse/app/components/modal/login.hbs

```diff
@@ -0,0 +1,56 @@
+<DModal
+  class="login-modal"
+  @bodyClass={{this.modalBodyClasses}}
+  @closeModal={{@closeModal}}
+  @flash={{this.flash}}
+  @flashType={{this.flashType}}
+  {{did-insert this.doTheThing}}
+>
+  <:body>
+    <PluginOutlet @name="login-before-modal-body" @connectorTagName="div" />
+    {{#if this.canLoginLocal}}
+      <div class="login-left-side">
+        <Modal::Login::WelcomeHeader @wavingHandURL={{this.wavingHandURL}} />
+        <Modal::Login::LocalLoginForm
+          @loginName={{this.loginName}}
+          @loginNameChanged={{this.loginNameChanged}}
+          @canLoginLocalWithEmail={{this.canLoginLocalWithEmail}}
+          @loginPassword={{this.loginPassword}}
+          @secondFactorMethod={{this.secondFactorMethod}}
+          @secondFactorToken={{this.secondFactorToken}}
+          @backupEnabled={{this.backupEnabled}}
+          @securityKeyAllowedCredentialIds={{this.securityKeyAllowedCredentialIds}}
+          @securityKeyChallenge={{this.securityKeyChallenge}}
+          @showSecurityKey={{this.showSecurityKey}}
+          @otherMethodAllowed={{this.otherMethodAllowed}}
+          @handleForgotPassword={{this.handleForgotPassword}}
+          @login={{this.login}}
+          @flashChanged={{this.flashChanged}}
+          @flashTypeChanged={{this.flashTypeChanged}}
+          @securityKeyCredentialChanged={{this.securityKeyCredentialChanged}}
+        />
+        <Modal::Login::Footer
```

Comment:
It is not ideal to render the `<div class="modal-footer">` within the body of the modal, but I am limited to doing this with the current css design of the login modal. I don't think we are necessarily limited by the current structure of the modal (body / footer) we are limited by the structure of the css / html implemented to make it look the way it does.

This would be worth raising with designers once these changes are through